src/components/cylc/Drawer.vue

```diff
@@ -59,89 +59,70 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <template v-slot:append>
       <div class="px-4 py-2 d-flex justify-center">
         <span class="text--secondary">
-          <strong v-if="$options.mode !== 'production'">{{ $options.mode.toUpperCase() }}</strong> {{ $t('App.name') }} {{ $options.version }}
+          <strong v-if="mode !== 'production'">{{ mode.toUpperCase() }}</strong> {{ $t('App.name') }} {{ pkg.version }}
         </span>
       </div>
     </template>
   </v-navigation-drawer>
 </template>
 
-<script>
+<script setup>
 import { nextTick, ref } from 'vue'
 import { useDisplay } from 'vuetify'
-import Header from '@/components/cylc/Header.vue'
+import CHeader from '@/components/cylc/Header.vue'
 import Workflows from '@/views/Workflows.vue'
 import { mdiHome, mdiGraphql } from '@mdi/js'
 import pkg from '@/../package.json'
 import { when } from '@/utils'
 import { useDrawer } from '@/utils/toolbar'
 
-export const initialWidth = 260
-export const minWidth = 150
```

Comment:
Ah this is the main snag in doing this I'm afraid. These 2 are exported and used in the tests

Reply:
Well it was fun trying

Reply:
I'm closing it then